### PR TITLE
[159] Add an example showing how to use DiagramWebSocketContainer

### PR DESCRIPTION
**Pull request by sbegaudeau**
Bug: https://github.com/eclipse-sirius/sirius-components/issues/159
Signed-off-by: Stéphane Bégaudeau <stephane.begaudeau@obeo.fr>

### Type of this PR 

- [ ] Bug fix
- [ ] New feature or improvement
- [x] Documentation
- [ ] Cleanup
- [ ] Test
- [ ] Build/Releng

### Issue(s)

...

### What does this PR do?

...

### Screenshot/screencast of this PR

...
 

### Potential side effects

...

### How to test this PR?

- [ ] Frontend Unit tests
- [ ] Backend Unit tests
- [ ] Cypress : please specify
- [x] Manual Test : please specify

### Checklist

- [x] I have read CONTRIBUTING carefully.
- [x] I have signed the [Contributor License Agreement](http://www.eclipse.org/legal/CLA.php).
- [x] All my commits are signed-off (`-s`) with my mail address of my Eclipse Account.
- [x] I have covered my changes by unit tests or integration tests or manual tests.
- [x] All tests pass.
- [ ] I have updated the documentation accordingly
- [ ] New React components are availables in storybook

### CODE REVIEW

File: sample/diagrams-only/frontend/src/index.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
```
Comment (pcdavid) [inline sample/diagrams-only/frontend/src/index.js:1]: Missing © header.

File: sample/diagrams-only/frontend/src/index.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+);
+
+export const ApolloGraphQLClient = new ApolloClient({
+  link: splitLink,
+  cache: new InMemoryCache(),
+});
+
+ReactDOM.render(
+  <ApolloProvider client={ApolloGraphQLClient}>
+    <React.StrictMode>
```
Comment (pcdavid) [inline sample/diagrams-only/frontend/src/index.js:129]: Remove StrictMode to workaround Apollo's limitation?

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -70,6 +70,21 @@ jobs:
...
+          echo '//npm.pkg.github.com/:_authToken=${NODE_AUTH_TOKEN}' >> .npmrc
+          npm ci
+          yalc add @eclipse-sirius/sirius-components
+          npm run build
+        working-directory: sample/diagrams-only/frontend
+
+      - name: Copy the diagrams-only sample frontend into its backend
+        run: |
+          mkdir -p "backend/sirius-web-test-diagrams-only-application/src/main/resources/static"
+          cp -R sample/diagrams-only/frontend/build/* backend/sirius-web-test-diagrams-only-application/src/main/resources/static
```
Comment (pcdavid) [inline .github/workflows/build.yml:87]: Am I missing something or is this backend not built?

**Review by pcdavid — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 3 above)
Comment (sbegaudeau) [reply .github/workflows/build.yml:87]: It is built with the other parts of the backend

**Review by pcdavid — APPROVED** [review]

**Pull request merged**